Zend/zend_virtual_cwd.c

```diff
@@ -1131,6 +1131,9 @@ CWD_API int virtual_file_ex(cwd_state *state, const char *path, verify_path_func
 		/* skip DRIVE name */
 		resolved_path[0] = toupper(resolved_path[0]);
 		resolved_path[2] = DEFAULT_SLASH;
+		if (path_length == 2) {
+			resolved_path[3] = 0;
```

Comment:
Nit:
```suggestion
			resolved_path[3] = '\0';
```
Makes it obvious we're dealing with a string.

Reply:
> Correct me if I'm wrong, can't this line read from uninitialized memory if we don't initialize resolved_path?

AFAICT: I don't think so. By the time line 1108 is reached we know that resolved_path must end in a NUL byte. (see above reasoning).
Therefore if len == 2, then resolved_path[2] must be a NUL byte. The while loop checks `!IS_SLASH(resolved_path[2])` which will be true because a NUL byte isn't a slash and the loop will be entered. Then the comparison against 0 will succeed and we will jump to `verify`.

> Makes it obvious we're dealing with a string.

Sure, I agree I'll change it soon and I'll take it into account for the future.

Reply:
@nielsdos Ah, yes you're right of course. Looks good then!